packages/ui/src/theme/css/component/alert.scss

```diff
@@ -1,25 +1,45 @@
 .amplify-alert {
   background-color: var(--amplify-components-alert-background-color);
-  padding: var(--amplify-components-alert-padding-vertical)
-    var(--amplify-components-alert-padding-horizontal);
+  padding-block: var(--amplify-components-alert-padding-block);
+  padding-inline: var(--amplify-components-alert-padding-inline);
 
   &[data-variation='info'] {
+    color: var(--amplify-components-alert-info-color);
     background-color: var(--amplify-components-alert-info-background-color);
   }
 
   &[data-variation='error'] {
+    color: var(--amplify-components-alert-error-color);
     background-color: var(--amplify-components-alert-error-background-color);
   }
 
   &[data-variation='warning'] {
+    color: var(--amplify-components-alert-warning-color);
     background-color: var(--amplify-components-alert-warning-background-color);
   }
 
   &[data-variation='success'] {
+    color: var(--amplify-components-alert-success-color);
     background-color: var(--amplify-components-alert-success-background-color);
   }
 }
 
-.amplify-alert-heading {
+.amplify-alert__icon {
+  font-size: var(--amplify-components-alert-icon-size);
+}
+
+.amplify-alert__heading {
+  color: inherit;
```

Comment:
I was under the impression that `color` prop is inherited by default. It seems that we have set `color` on `Heading` and was wondering if we should remove that and let the color inherit itself on `Heading`

Reply:
You are correct, I actually added this in before I changed the heading from a `<Heading>` to a `<View>`. This style was needed if we used a Heading, but not if we used a View.